DEVELOPER_GUIDE.md

```diff
@@ -47,6 +47,18 @@ The following information will help in getting up and running.
   cargo install --force cargo-make
   ```
 
+- [**Packages**]()
+
+  Use your preferred package manager to install the following packages:
```

Comment:
We should probably make it distribution specific, something along this lines:

```
On Ubuntu Linux:  apt install build-essential pkg-config libssl-dev
On Alpine Linux:  apk add openssl-dev pkgconfig build-base
```

Reply:
+1 it will be different on Windows

There's no need to include all sections, but maybe a single new "Linux" section is a good start.

Reply:
I don't exactly follow, should I include both each of the ones @reta mentioned as well as Windows? Or just one for linux and one for Windows?

Reply:
Up to you. If you know the instructions for both Linux and Windows I would add both. If you only know one, include that. But either way I think both of us are saying there should be sections for each.